sidebars.ts

```diff
@@ -10,22 +10,62 @@ import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
 
  Create as many sidebars as you want.
  */
+
+let migraUruguayLinks: string[] = [
+  'migra/panama/visa-de-transito',
+  'migra/uruguay/consulado-uruguay-en-cuba/legalizacion-documentos',
+  'migra/uruguay/consulado-uruguay-en-cuba/reunificacion-familiar'
+];
+
+let migraPanamaLinks: string[] = [
+  'migra/panama/visa-de-transito'
+]
+
 const sidebars: SidebarsConfig = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
-
+  //migrateSidebar: [{type: 'autogenerated',dirName: 'migra'}],
+  //businessSidebar: [{type: 'autogenerated', dirName: 'business'}],
   // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
-    'intro',
-    'hello',
+
+  migrateSidebar: [
     {
       type: 'category',
-      label: 'Tutorial',
-      items: ['tutorial-basics/create-a-document'],
+      label: 'Migración',
+      description: "Todos los temas migratorios en un sólo lugar",
+      items: ['migra/intro',   
+        {type: 'category',label: 'Panamá', items: migraPanamaLinks},
+        {type: 'category',label: 'Uruguay', items: migraUruguayLinks}
+      ],
     },
   ],
-   */
+  businessSidebar: [
+    {
+      type: 'category',
+      label: 'Negocios',
+      items: ['business/intro',
+        {type: 'category',label: 'Bitcoin',items: ['business/bitcoin/intro']},
+        {type: 'category',label: 'Monedas',items: ['business/currencies/intro']}
+      ]
+    }
+  ],
+  ocioSidebar: [
+    {
+      type: 'category',
+      label: 'Ocio',
+      items: ['ocio/intro',
```

Comment:
And change this for the term on english. 

The idea is to have a homogeneous route system.